python/examples/test-py-arch.py

```diff
@@ -0,0 +1,92 @@
+# Example Python Arch plugin written in Python
+# ============================================
+#
+#  -- pancake @ nopcode.org
+#
+# The r2lang.plugin function exposes a way to register new plugins
+# into the RCore instance. This API is only available from RLang.
+# You must call with with '#!python test.py' or 'r2 -i test.py ..'
```

Comment:
Better use -I to load earlier

the resr of comments can be removed

Reply:
I copied that from the other examples. Would be nice if -a would run after the plugin is loaded, but no difference between -i and -I for that.